src/levanter/models/backpack.py

```diff
@@ -241,21 +245,23 @@ def init(
             key=k_mlp,
             use_bias=config.use_bias,
         )
+        ln_2 = hnn.LayerNorm.init(config.Senses, eps=config.layer_norm_epsilon)
```

Comment:
i think this should be Embed?

Reply:
(you want to normalize over the embed dimension, sort of like softmax(x, axis=Pos) rather than softmax(x, axis=Head) or something)